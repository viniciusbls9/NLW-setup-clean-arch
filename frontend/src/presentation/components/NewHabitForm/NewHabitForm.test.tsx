import { describe, test, expect } from 'vitest';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { NewHabitForm } from '.';

describe('Header Component', () => {
  test('Should render correctly modal component', () => {
    const { container } = render(<NewHabitForm />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
