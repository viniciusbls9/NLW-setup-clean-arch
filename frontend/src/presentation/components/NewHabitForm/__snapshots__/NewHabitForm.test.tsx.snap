// Vitest Snapshot v1

exports[`Header Component > Should render correctly modal component 1`] = `
<form
  class="w-full flex flex-col mt-6"
>
  <label
    class="font-semibold leading-tight"
    for="title"
  >
    Qual seu comprometimento
  </label>
  <input
    class="p-4 rounded-lg mt-3 bg-zinc-800 text-white placeholder:text-zinc-400"
    id="title"
    placeholder="Exercícios, dormir bem, etc..."
    type="text"
  />
  <label
    class="font-semibold leading-tight mt-4"
    for=""
  >
    Qual a recorrência?
  </label>
  <button
    class="mt-6 rounded-lg p-4 flex items-center justify-center gap-3 font-semibold bg-green-600 hover:bg-green-500"
    type="submit"
  >
    <svg
      fill="currentColor"
      height="20"
      viewBox="0 0 256 256"
      width="20"
      xmlns="http://www.w3.org/2000/svg"
    >
      <rect
        fill="none"
        height="256"
        width="256"
      />
      <polyline
        fill="none"
        points="216 72 104 184 48 128"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="24"
      />
    </svg>
    Confirmar
  </button>
</form>
`;
